cdc/pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -379,17 +330,6 @@ func (o *createChangefeedOptions) run(ctx context.Context, cmd *cobra.Command) e
 		return err
 	}
 
-	if !o.commonChangefeedOptions.noConfirm {
-		currentPhysical, _, err := o.pdClient.GetTS(ctx)
-		if err != nil {
-			return err
-		}
-
-		if err := confirmLargeDataGap(cmd, currentPhysical, o.startTs); err != nil {
```

Comment:
The `confirmLargeDataGap` may be useful for us. It is used to prevent OOM.. Suggest to keep it.